artifactory/commands/transferinstall/transferinstall.go

```diff
@@ -0,0 +1,36 @@
+package transferinstall
+
+import (
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+)
+
+const (
+	groovyFileName  = "dataTransfer.groovy"
+	jarFileName     = "data-transfer.jar"
+	dataTransferUrl = "https://releases.jfrog.io/artifactory/jfrog-releases/data-transfer"
+	libDir          = "lib"
+)
+
+type InstallTransferCommand struct {
+	InstallPluginCommand
+}
+
+var transferPluginFiles = PluginFiles{
+	FileItem{groovyFileName},
+	FileItem{libDir, jarFileName},
+}
+
+func NewDataTransferInstallFileManager() *PluginInstallManager {
+	manager := NewArtifactoryPluginInstallManager(transferPluginFiles)
+	return manager
+}
+
+func (tic *InstallTransferCommand) CommandName() string {
+	return "rt_transfer_install"
```

Comment:
Since this command does not implement the `Command` interface - 
```
type Command interface {
	// Runs the command
	Run() error
	// Returns the Server details. The usage report is sent to this server.
	ServerDetails() (*config.ServerDetails, error)
	// The command name for the usage report.
	CommandName() string
}
```
there's no need for you to implement this function. I suggest you also discuss with @RobiNino the reason for having some of our commands implementing this interface.

Reply:
as explained above InstallPluginCommand implements the Run() and ServerDetails() and the specific InstallDataTransferPluginCommand provides the last method CommandName(). therefore it is implementing the interface and I can see in testing that the commands.Exec(installCmd) logs that usage report are sent